common/test/package.scala

```diff
@@ -59,7 +60,11 @@ trait TestSettings {
   ContentApiClient.jsonClient._http = toRecorderHttp(ContentApiClient.jsonClient._http)
 }
 
-trait ConfiguredTestSuite extends ConfiguredServer with ConfiguredBrowser with ExecutionContexts {
+trait WithTestWsClient {
+  lazy val wsClient = NingWSClient(NingWSClientConfig(maxRequestRetry = 0))
```

Comment:
Two things here

- NingWsClient will need to be replaced by AhcWsClient when bumping to play 2.5. There's no way around it.
- When creating a wsClient yourself, you need to handle its destruction.

Here's an extract of the docs

> If you create a WSClient manually then you must call client.close() to clean it up when you’ve finished with it.

you can find it here https://www.playframework.com/documentation/2.5.x/ScalaWS#Using-WSClient